style.css

```diff
@@ -0,0 +1,139 @@
+/* Universal styles */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Times New Roman", Times, serif;
+}
+
+.parent-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.centered-div {
+  width: 375vw;
+  height: 100vh;
+}
+
+/* Styling the Navigation bar */
+.nav-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav-container li {
+  list-style: none;
+  margin: 8px 8px 8px 8px;
+  justify-content: left;
+}
+
+nav .logo {
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  margin-top: 4px;
+  margin-left: 12px;
+  letter-spacing: normal;
+  text-align: center;
+  padding: 10px 12px;
+}
+
+.menu-bars {
+  color: #7f8cff;
+  margin-right: 24px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+}
+
+nav .menu-bar {
+  width: 20px;
+  background: #7f8cff;
+  border: 2px solid #7f8cff;
+}
+
+/* Styling the background image */
+main {
+  background-image: url("images/header-shapes_mobile\ \(1\).jpg");
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
```

Comment:
- [ ] Kindly make sure your background image is covering a whole area of the container in order to look the same as the provided Figma design. :wink: 

   - ### YOUR DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/205569023-dccafcc4-eca3-4114-b021-f1077d2d74eb.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/205569168-9af2019b-e7c7-4027-bb56-5da914900356.png)

Reply:
Hi @levy002, I made all the changes that you recomendaded but when I pushed the changes I had a lighthouse error. Any idea what could be the problem.